config/routes.rb

```diff
@@ -316,7 +321,7 @@ def patch(*)
             end
           end
 
-          resources :web_hooks
+          resources :web_hooks, only: %i[index create show edit update destroy]
```

Comment:
For a second I was like "aren't those all the actions??" 😄 (spoiler: there's no `new` 😛)

Reply:
Ya, welcome to my day. I prefer `only` over `except` though.